doc/clientApp/clientApp_all.tex

```diff
@@ -193,4 +193,25 @@ \subsection{Archive experiment reconstruction}
 \begin{itemize}
 	\item Clicking the button on the archive page of the specific experiment.
 	\item Using the URL with the parameter  "archive=XXX". Useful to share results.
-\end{itemize}
\ No newline at end of file
+\end{itemize}
+
+\subsection{Interactive world map}
+Demos working with satellite images often need that the user selects a zone on a map which can be interactively zoomed in and out. The zone is typically defined by a polygon drawn over the map.
+%
+The data format is \href{https://geojson.org/}{GeoJSON}, since its specification has been standardized by the \href{https://www.ietf.org/}{IETF} as \href{https://www.rfc-editor.org/rfc/rfc7946}{RFC 7946}.
+
+The \href{https://www.mapbox.com/}{Mapbox} company offers several map services under the \emph{pay-as-you-go} modality, including the interactive widget Mapbox GL JS which can be included in websites. The control is \href{https://docs.mapbox.com/mapbox-gl-js/api/}{properly documented}, including \href{https://docs.mapbox.com/help/tutorials/?product=Mapbox+GL+JS}{tutorials} and \href{https://docs.mapbox.com/mapbox-gl-js/example/}{examples}.
+
+To add the map on a website one needs to obtain an access token, include the mapbox-gl JS libraries and instantiate a new mapboxgl.Map object. Please look at the Mapbox's \href{https://docs.mapbox.com/mapbox-gl-js/example/simple-map/}{example}. The Map object has a programatic \href{https://docs.mapbox.com/mapbox-gl-js/api/map/}{JS API} several methods and properties to easily interact with it.
```

Comment:
no longer relevant
```suggestion
```

Reply:
I've added information about Maplibre, explaining that it's our preferred option.
However, there's no reason to hide the fact that it's a fork from Mapbox, specially if they provide useful examples from a very similar, if not the same, base of code.